x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/TraceSummary.tsx

```diff
@@ -0,0 +1,255 @@
+}
+const Duration: React.FunctionComponent<DurationProps> = ({
+  duration,
+  totalDuration
+}) => {
+  totalDuration = totalDuration || duration;
+  const isOver100 = duration > totalDuration;
+  const percentOfTrace = isOver100
+    ? '>100%'
+    : asPercent(duration, totalDuration, '');
+  const percentOfTraceText = i18n.translate(
+    'xpack.apm.transactionDetails.percentOfTrace',
+    {
+      defaultMessage: '{value} of trace',
+      values: { value: percentOfTrace }
+    }
+  );
+
+  return (
+    <Item>
+      <EuiToolTip
+        content={i18n.translate('xpack.apm.transactionDetails.durationLabel', {
+          defaultMessage: 'Duration'
+        })}
+      >
+        <>{asTime(duration)}</>
+      </EuiToolTip>{' '}
+      {isOver100 ? (
+        <EuiToolTip
+          content={i18n.translate(
+            'xpack.apm.transactionDetails.percentOfTraceLabelExplanation',
+            {
+              defaultMessage:
+                'The % of trace exceeds 100% because this transaction takes longer than the root transaction.'
+            }
+          )}
+        >
+          <>({percentOfTraceText})</>
+        </EuiToolTip>
+      ) : (
+        `(${percentOfTraceText})`
+      )}
+    </Item>
+  );
+};
+
+const MethodText = styled('span')`
+  font-weight: bold;
+`;
+
+interface HttpStatusBadgeProps {
+  status?: number;
+}
+const HttpStatusBadge: React.FunctionComponent<HttpStatusBadgeProps> = ({
+  status
+}) => {
+  let color = 'default';
+
+  if (!status) {
+    return null;
+  } else if (status < 300) {
+    color = theme.euiColorVis0;
+  } else if (status >= 300 || status < 400) {
+    color = theme.euiColorVis5;
+  } else if (status >= 400 || status < 500) {
+    color = theme.euiColorVis7;
+  } else if (status >= 500) {
+    color = theme.euiColorVis2;
+  }
+
+  return (
+    <EuiToolTip
+      content={i18n.translate('xpack.apm.transactionDetails.statusCode', {
+        defaultMessage: 'Status code'
+      })}
+    >
+      <EuiBadge color={color}>{status}</EuiBadge>
+    </EuiToolTip>
+  );
+};
+
+const HttpInfoBadge = styled(EuiBadge)`
+  max-width: 250px;
+`;
+
+interface HttpInfoProps {
+  transaction: Transaction;
+}
+const HttpInfo: React.FunctionComponent<HttpInfoProps> = ({ transaction }) => {
+  const method = (
+    idx(transaction, _ => _.http.request.method) || ''
+  ).toUpperCase();
+  const isRumAgent = isRumAgentName(transaction.agent.name);
+  const url = isRumAgent
+    ? idx(transaction, _ => _.transaction.page.url) || NOT_AVAILABLE_LABEL
+    : idx(transaction, _ => _.url.full) || NOT_AVAILABLE_LABEL;
+  const status = idx(transaction, _ => _.http.response.status_code);
+
+  return (
+    <>
+      <HttpInfoBadge title={undefined}>
+        <EuiToolTip
+          content={i18n.translate(
+            'xpack.apm.transactionDetails.requestMethodLabel',
+            {
+              defaultMessage: 'Request method'
+            }
+          )}
+        >
+          <MethodText>{method}</MethodText>
```

Comment:
Perhaps simply use `<strong>{method}</strong>` ?

Reply:
Or maybe that's abuse to use semantic tags for styling. I'm okay with either

Reply:
Using an explicit styled component is probably safer and more "semantic", but I'll just use `string` because it's less code.